two_factor/plugins/phonenumber/__init__.py

```diff
@@ -0,0 +1,4 @@
+import django
+
+if django.VERSION <= (3, 2):
```

Comment:
Is this line (and the one for yubikey) necessary? I.e. will Django 4.0 start spitting out warnings if we don't put in this if statement